projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1451,8 +1451,13 @@ public function get_author( $author, $show_email_and_ip = false ) {
 				$nice       = $user->user_nicename;
 			}
 			if ( defined( 'IS_WPCOM' ) && IS_WPCOM && ! $is_jetpack ) {
-				$active_blog = get_active_blog_for_user( $id );
-				$site_id     = $active_blog->blog_id;
+				$active_blog = apply_filters( 'wpcom_pre_get_active_blog_for_user', false, $id );
```

Comment:
Could you add a docblock, to make our parser happy?

```suggestion
				/**
				 * Allow customizing the blog ID returned with the author in WordPress.com REST API queries.
				 *
				 * @since $$next-version$$
				 *
				 * @module json-api
				 *
				 * @param bool|int $active_blog  Blog ID, or false by default.
				 * @param int      $id           User ID.
				 */
				$active_blog = apply_filters( 'wpcom_pre_get_active_blog_for_user', false, $id );
```

Maybe we could also update the filter name to make it clearer where the filter could be used?

Reply:
Done. Do you have any suggestions for the filter name?

Reply:
Maybe we could mention something about this being used in the API, and author-related? Something like `wpcom_api_pre_get_active_blog_author`? What do you think?

Reply:
Sounds good to me!